### DISCUSSION TITLE
Seeds.. lots of them.

### DISCUSSION THREAD

**Discussion by burning-eggs** [jeremyckahn/farmhand#155] [Ideas]
# Lists of seeds I think good addition

Aconite seeds
Alkanet seeds
Aloe seeds
Amla seeds
Andrographis seeds
Angelica seeds
Arnica seeds
Artichoke seeds
Aragula seeds
Ashwagandha seeds

## I will add more onto these later??? probably

**Comment by jeremyckahn** [comment]
Yes to more seeds! Some of these are a little obscure, and we should try to keep Farmhand's appeal as wide as we reasonably can. That said, I'd like to pare this down to some of the more recognizable seeds:

  - Aloe
  - Artichoke
  - Arugula

I could be convinced to add Arnica, but I think the others a little more commonplace and should therefore be implemented first.

**Reply by burning-eggs** [reply]
sure, feel free to ping me on discord @flying-dojo if you want me to make any seed art

**Reply by jeremyckahn** [reply]
Ok! Just for reference, you can look at this Asparagus commit as a guide for implementing a crop in case you'd like to do it yourself: https://github.com/jeremyckahn/farmhand/commit/a24b11edf7f685812d2eb0fe2d7dff2016b84869

(You can ignore the "sprout" files here, as that crop stage is not currently supported.)
